cng/ecdh.go

```diff
@@ -126,22 +126,17 @@ func GenerateKeyECDH(curve string) (*PrivateKeyECDH, []byte, error) {
 	// GenerateKeyECDH returns the public key as as byte slice.
 	// To get it we need to export the raw CNG key blob
 	// and prepend the encoding prefix.
```

Comment:
We aren't doing this anymore, right?

```suggestion
	// To get it we need to export the raw CNG key bytes.
```